src/lib.rs

```diff
@@ -30,7 +30,7 @@
 //! feature.
 #![cfg_attr(feature="clippy", feature(plugin))]
 #![cfg_attr(feature="clippy", plugin(clippy))]
-#![cfg_attr(feature="clippy", deny(clippy))]
+#![cfg_attr(feature="clippy", warn(clippy))]
```

Comment:
Does it not work to leave `deny` here but `warn` on the one function? It's not a big deal anyway.